src/app/games/game-summaries/game-summary/game-summary.component.ts

```diff
@@ -22,7 +22,7 @@ export class GameSummaryComponent implements OnChanges {
   /**
    * The percentage to display for completion.
    */
-  total: number;
+  total: number = -1;
```

Comment:
```suggestion
  total: number = null;
```
Would this work by chance? There might be another workaround that could  be added that would remove displaying a percentage/number.

Reply:
I think it does, at least for now.